### ISSUE TITLE
Can't Export Localizations because of this framework

### ISSUE THREAD

**Issue by qizh** [google/GoogleSignIn-iOS#305]
It fails with the following error:
```
Undefined symbols for architecture arm64:
  "_OBJC_CLASS_$_OIDAuthState", referenced from:
      objc-class-ref in GoogleSignIn.o
  "_OBJC_CLASS_$_OIDAuthorizationRequest", referenced from:
      objc-class-ref in GoogleSignIn.o
  "_OBJC_CLASS_$_OIDAuthorizationService", referenced from:
      objc-class-ref in GoogleSignIn.o
  "_OBJC_CLASS_$_OIDIDToken", referenced from:
      objc-class-ref in GoogleSignIn.o
  "_OBJC_CLASS_$_OIDServiceConfiguration", referenced from:
      objc-class-ref in GoogleSignIn.o
  "_OBJC_CLASS_$_OIDURLQueryComponent", referenced from:
      objc-class-ref in GoogleSignIn.o
  "_OIDOAuthErrorResponseErrorKey", referenced from:
      +[GIDEMMSupport handleTokenFetchEMMError:completion:] in GoogleSignIn.o
  "_OIDOAuthTokenErrorDomain", referenced from:
      ___53-[GIDGoogleUser refreshTokensIfNeededWithCompletion:]_block_invoke.20 in GoogleSignIn.o
  "_OIDResponseTypeCode", referenced from:
      -[GIDSignIn authenticateInteractivelyWithOptions:] in GoogleSignIn.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

It works when making a regular or release build, but fails when using `Product → Export Localizations…`

**Environment**
- Xcode v14.3
- GoogleSignIn-iOS v7.0.0 added with SPM

Is there a workaround to make it work?

**Comment by mdmathias** [comment]
Sorry you're having trouble. I just created a simple project that pulls in GoogleSignIn-iOS via SPM and wasn't able to reproduce this issue with Xcode 14.3. I also tested in our [ObjC and Swift sample apps](https://github.com/google/GoogleSignIn-iOS/tree/main/Samples/) and didn't see the problem there either.

Can you add more about how things are set up in your project?

**Comment by qizh** [comment]
I'm adding `GoogleSignIn-iOS` to the project's framework target that is used by multiple app targets of the same project. 
I've tried to reproduce the structure I have in a new test project and everything works well in it 🤦🏻‍♂️
So there's something wrong with my project configuration I have to find and fix.
Sorry for wasting your time.

**Comment by cjwirth** [comment]
I'm having these same issues but I'm just trying to build normally. Did you ever figure out what your issue was? 

My app is also broken up into a few frameworks, and it's kinda set up like this:

```
.
└── AppTarget
    ├── OnboardingFramework
    │   └── GoogleSignIn
    └── LoginFramework
        └── GoogleSignIn
```

**Comment by qizh** [comment]
@cjwirth, I've resolved my issue, but I don't recall how I did it.
I guess updating to the latest version of Xcode resolved the localization export issues.

Though my frameworks setup was different than yours. I have additional shared framework which contains the `GoogleSignIn` and few other dependencies I need in most of my other frameworks. Then this shared framework was added to each other frameworks like yours `OnboardingFramework` and `LoginFramework`.

But I don't remember exactly, this is just what came up to my mind right now.
